documentation/user-consent.md

```diff
@@ -29,11 +29,12 @@ This document captures guidelines for Operator platform to handle user consents.
 
 ## Introduction
 
-Operator platform implementing CAMARA should be built with a privacy-by-design approach to fully comply with data protection regulations like the [GDPR regulation](https://gdpr-info.eu/) in Europe, which provides a great level of protection for user's privacy. This means that an API that processes Personal Information needs a user consent. Consents are given by users to legal entities to process personal data under a specific purpose.
+Some APIs process personal information and require a “legal basis” to do so (e.g. “legitimate interest”, “contract”, “consent”, etc). Telco Operator Exposure platforms implementing CAMARA should be built with a privacy-by-design approach to fully comply with data protection regulations, such as the [GDPR regulation](https://gdpr-info.eu/) in Europe, to protect user privacy. This means that a CAMARA API exposed to Capability Consumers that processes personal data may require user consent (explicit user opt-in), depending on the "legal basis" for processing that data. This consent is given by users to legal entities to process personal data under a specific purpose.
 
-This document captures guidelines for Operator platform to handle user consents to comply with GDPR or equivalent requirements in an easy way, introducing the concept of purpose in OpenID Connect. Even being defined based on concepts that maps to GDPR regulation, proposed solution and concepts are generic enough to be used by Operators on any country.
+As per CAMARA Commonalities ["Authentication and Authorization Concept for Service APIs"](https://github.com/camaraproject/WorkingGroups/blob/main/Commonalities/documentation/CAMARA-AuthN-AuthZ-Concept.md), CAMARA API access will be securitized using [OpenID Connect](https://openid.net/connect/) on top of OAuth 2.0 protocol. This document captures guidelines for Operator Exposure platform to manage CAMARA API access and user consent to comply with GDPR or equivalent requirements in an easy way, introducing the concept of purpose in OpenID Connect. Even being defined based on concepts that maps to GDPR regulation, proposed solution and concepts are generic enough to be used by Operators on any country.
```

Comment:
To match with earlier change "This document _defines_ guidelines..."

Reply:
Fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/13/commits/0162b0ae7a5045d187219ad55f0dd2a2dd45f9d1